phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -537,6 +536,7 @@ private boolean isMatchingMutation(Mutation expected, Mutation actual, int itera
                     byte[] dataKey = indexMaintainer.buildDataRowKey(new ImmutableBytesWritable(expected.getRow()), viewConstants);
                     String errorMsg = "Missing cell (in iteration " + iteration + ") " + Bytes.toString(family) + ":" + Bytes.toString(qualifier);
                     logToIndexToolOutputTable(dataKey, expected.getRow(), getTimestamp(expected), getTimestamp(actual), errorMsg);
+                    verificationPhaseResult.mismatchingNumberOfCellsCount++;
```

Comment:
This is the missing cell case and the one below is extra cell case. Should we separate them to know better?

Reply:
They have the same underlying cause mostly. What do you think @kadirozde ?

Reply:
I think there is value in having separate counters for missing and extra cells. It will give us more info.